components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/package.json

```diff
@@ -22,7 +22,8 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "clean": "rimraf dist",
-    "dev": "NODE_ENV=development webpack -d --config webpack.config.js --watch"
+    "dev": "NODE_ENV=development webpack -d --config webpack.config.js --watch",
+    "build-dev": "webpack -d --config webpack.config.js --watch --progress"
```

Comment:
Why have both `dev` and `build-dev`?

Reply:
Are we using webpack dev server here? If not, then no point of having `--watch` flag.

Reply:
Better to use this `dev` script https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/package.json#L9